src/renderer/store/modules/settings.js

```diff
@@ -226,6 +226,7 @@ const state = {
   hideLabelsSideBar: false,
   hideChapters: false,
   showDistractionFreeTitles: false,
+  keyboardShortcuts: false,
```

Comment:
Maybe this should be set to true as some users are already using keyboard shortcuts and will probably open an issue thinking that shortcuts are broken, not realizing they have to enable it

Reply:
Definitely should be enabled by default, as the keyboard shortcuts have existed for a long time and most users want them.

A better option would be to name the setting `Disable Keyboard Shortcuts`, not only does that clearly indicate to the user that the default is to have the keyboard shortcuts enabled, but it would also make the code clearer as you would have `disableKeyboardShortcuts`, instead of the ambiguously named `keyboardShortcuts` value, that doesn't immediately tell you that it is a boolean.